### ISSUE TITLE
Staging Review finding: Phone number error not announced to screen readers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124606]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** VA Forms 21P-530a
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When there is an error for the phone number field and group, the error is not announced when the phone number field has focus. Additionally, the field is not identified as being `invalid` due to `aria-invalid="false"` on a field in error. 

Due to this, screen reader users may not be able to determine that the field is in error and may not be able to proceed. 

<details><summary>Link, screenshot or steps to recreate</summary>

- using Jaws in Chrome
- Navigate to step 2 of 6 Burial Benefits recipient
-  Enter a Full name
- TAB past phone number fields triggering validation on the phone number group
- Shift + TAB back into the phone number field
- Note screen reader announcement

`<input class="usa-input" id="inputField" type="tel" aria-invalid="false" inputmode="tel" autocomplete="tel" required="" part="input">`

<img width="398" height="382" alt="phone error" src="https://github.com/user-attachments/assets/cc81569e-84b7-4ce4-b7e5-3eb5c58932a8" />

</details>

## Recommended action

Ensure that the field is identified as in error progammatically using `aria-invalid="true"` and ensure that the error message is associated to the field in error, so that screen readers announce the error when the field has focus. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
